src/components/WalletSelect.vue

```diff
@@ -159,7 +159,7 @@ export default {
         </div>
       </div>
       <div id="wallet-select-header" class="flex items-right">
-        <h1>Select Wallet</h1>
+        <h1>Select Crypto Wallet</h1>
```

Comment:
I'm going to override them on this: sorry, I didn't notice it before.

please keep as "Select Wallet"

Bitcoiners get cranky when you call it Crypto